src/serializers/pets-serializer.js

```diff
@@ -2,7 +2,7 @@ import { Serializer as JsonApiSerializer } from 'jsonapi-serializer';
 import _ from 'lodash';
 
 import { serializerOptions } from 'utils/jsonapi';
-import { openapi } from 'utils/load-openapi';
+import { openapi } from 'utils/load-openapi-pet';
 import { paginate } from 'utils/paginator';
 import { apiBaseUrl, resourcePathLink, paramsLink } from 'utils/uri-builder';
 
```

Comment:
You should remove those files which are not needed.

Reply:
I'll delete these ones, I am implementing one endpoint at a time, and the structure is already in place, once every piece would go through my hand for Books resource, I would delete all unnecessary ones, these are there because still Patch and general Get have to be coded.